extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/AstroHandlerFactory.java

```diff
@@ -29,20 +29,23 @@
  * @author Gerhard Riegler - Initial contribution
  */
 public class AstroHandlerFactory extends BaseThingHandlerFactory {
-    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.union(SunHandler.SUPPORTED_THING_TYPES,
+
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.union(SunHandler.SUPPORTED_THING_TYPES,
             MoonHandler.SUPPORTED_THING_TYPES);
-    private static final Map<String, AstroThingHandler> astroThingHandlers = new HashMap<String, AstroThingHandler>();
+    private static final Map<String, AstroThingHandler> astroThingHandlers = Maps.newHashMap();
```

Comment:
If you just want to instantiate a HashMap, please to it without the use of an external library.